src/Slider/Slider.js

```diff
@@ -151,75 +152,91 @@ const getStyles = (props, context, state) => {
   return styles;
 };
 
-class Slider extends Component {
+class Slider extends React.Component {
   static propTypes = {
     /**
      * The default value of the slider.
      */
     defaultValue: valueInRangePropType,
+
     /**
      * Describe the slider.
      */
-    description: PropTypes.string,
+    description: React.PropTypes.string,
+
     /**
      * Disables focus ripple if set to true.
      */
-    disableFocusRipple: PropTypes.bool,
+    disableFocusRipple: React.PropTypes.bool,
+
     /**
      * If true, the slider will not be interactable.
      */
-    disabled: PropTypes.bool,
+    disabled: React.PropTypes.bool,
+
     /**
      * An error message for the slider.
      */
-    error: PropTypes.string,
+    error: React.PropTypes.string,
+
     /**
      * The maximum value the slider can slide to on
      * a scale from 0 to 1 inclusive. Cannot be equal to min.
      */
     max: minMaxPropType,
+
```

Comment:
Could you revert this change? See https://github.com/callemall/material-ui/pull/4073 for more details.